docs/specs/sections/protocol.tex

```diff
@@ -0,0 +1,79 @@
+
+The workflow is depicted in Figure \ref{fig:protocol}, and described with full details as follows.
+
+\begin{figure}[h]
+	\centering
+		\includegraphics[width=390pt,draft=false]{images/protocol.eps}
+	\caption{Overview of the protocol messages exchanged between the user, the Dusk Network, and the SP.}
+	\label{fig:protocol}
+\end{figure}
+
+\begin{enumerate}
+	\item (\textbf{user}) $\mathsf{send\_license\_req}$ : Compute a license stealth address $(\lpk, R)$ belonging to the user, using the user's own public key, and also an additional key $\ksym_{\License} = \hb(\lsk)G$, by computing first the user's $\lsk$. Then, compute the request stealth address $(\rpk, R)$ and $\kdh$ using the SP's public key. And finally send the following request to the network:
+
+
+		$$\Request = ((\rpk, R), \enc, \nonce)$$
+
+	where 
+
+		$$\enc = \Enc_{\kdh} ((\lpk, R)||\ksym_{\License}; \nonce)$$
+
+
+	\item (\textbf{SP}) $\mathsf{get\_license\_req}$ : Continuously check the network for incoming license requests.
```

Comment:
I would add how this is done, that is, the exact equation with the stealth address that needs to be checked.